src/Enterspeed.Source.UmbracoCms.V10/build/Enterspeed.Source.UmbracoCms.V9.targets

```diff
@@ -0,0 +1,27 @@
+﻿<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
This file should properly be renamed to reflect version :)

Reply:
Done :)